### PR TITLE
Adding a 'makeyourown' kernel tutorial, and a 'varying biofouling parameters' tutorial.

### PR DESCRIPTION
**Pull request by michaeldenes**
Pull request to add two new tutorials to the examples folder.

### CODE REVIEW

File: docs/examples/example_Croatian_fisheries.ipynb
Comment (erikvansebille) [inline docs/examples/example_Croatian_fisheries.ipynb:1]: Nice notebook! A few comments
1. Perhaps the title is a bit long? That now runs over multiple lines in the documentation
2. Is the `os.chdir('../../')` in cell 1 necessary?

File: docs/examples/example_make_your_own_kernel.ipynb
Comment (erikvansebille) [inline docs/examples/example_make_your_own_kernel.ipynb:1]: Nice example. A few comments:
1. Should the title not be 'Add your own behaviour kernels'?
2. Again, is the `os.chdir('../../')` in cell 1 necessary? If people have their $PYTHONPATH set-up correctly (and/or installed via conda) this should all work?
3. The print in cell 4 creates one long line, which is not nicely viewable in the docs. Print each kernel and flag on a separate line?
4. Do something about the warning in cell 17 (e.g. change the naming of ocean_u and ocean_v in each of the kernels?)

File: docs/examples/example_Croatian_fisheries.ipynb
Comment (michaeldenes) [reply docs/examples/example_Croatian_fisheries.ipynb:1]: Updated now, the import os is only necessary on my machine, now removed!

File: docs/examples/example_make_your_own_kernel.ipynb
Comment (michaeldenes) [reply docs/examples/example_make_your_own_kernel.ipynb:1]: I've now fixed all of the above in [b77f7e5]

**Review by erikvansebille — APPROVED** [review]

**Pull request merged**